ui/apps/platform/src/Containers/VulnMgmt/VulnMgmt.fragments.js

```diff
@@ -243,7 +243,7 @@ export const VULN_CVE_LIST_FRAGMENT = gql`
 `;
 
 export const IMAGE_CVE_LIST_FRAGMENT = gql`
-    fragment cveFields on ImageVulnerability {
+    fragment imageCVEFields on ImageVulnerability {
```

Comment:
I think we need to use this new name in the file `ui/apps/platform/src/Containers/VulnMgmt/List/Cves/VulnMgmtListCves.js`  above, at line 321.

Reply:
Oh, good catch